CONTRIBUTING.md

```diff
@@ -110,6 +202,143 @@ merge. You'll probably get some feedback from these fine folks which helps to
 make the project that much better. Respond to the feedback and work with your
 reviewer(s) to resolve any issues.
 
+Reviewers are responsible for ensuring that each merged PR's commit message
+conforms to [conventional commits](https://conventionalcommits.org). This may
+mean editing the commit message when you merge the pull request. Alternately,
+the reviewer may ask the pull request submitter to amend the commit message of
+their initial commit.
+
+## Releases
+
+This repository includes a set of tools for releasing gems. Only maintainers
+can perform releases.
+
+### Normal release process
+
+Releases are normally performed using GitHub Actions.
+
+ 1. Wait for the CI checks on the latest master branch commit to succeed. The
+    release scripts will not run unless the build is green.
+ 2. In the GitHub UI, go to the `Actions` tab, select the
+    `Open release request` workflow, and run the workflow manually using the
+    dropdown in the upper right.
+     *  Releases must be run from the master branch.
+     *  For the `Gems to release` field, enter the gem names to release, space
+        delimited. You can also leave the field blank, and the script will find
+        all the gems that have had conventional-commit-tagged changes since
+        their last release.
+ 3. The workflow will analyze the conventional commit messages for the gems to
+    release, and will open a _release pull request_. This pull request will
+    include the appropriate changes to each gem's version constants, and an
```

Comment:
Does this bit mean we don't need to bump the `VERSION` constants manually? If so, ❤️

Reply:
Yes, it will bump the version based on its understanding of the semver analysis of the commits. (But if you don't like the version it chooses, you can change it before merging.